prisma/schema.prisma

```diff
@@ -12,12 +12,24 @@ generator client {
   previewFeatures = ["referentialIntegrity"]
 }
 
-model Card {
-  id              Int       @default(autoincrement()) @id
+model DxDeck {
```

Comment:
The Dx prefix on Card and Deck was just for your experimentation, right?

Reply:
Yes! I'll change these to simply "Card" and "Deck" and re-push.

Reply:
Fixed in: 589075a2bb32f1f0d01892b214e0a8262e1a3ff6